storage-operator/deploy/crds/storage_v1alpha1_volume_crd.yaml

```diff
@@ -67,6 +67,56 @@ spec:
           - storageClassName
           type: object
         status:
+          properties:
+            failedContext:
+              properties:
+                errorCode:
+                  enum:
+                  - 0
+                  - 1
+                  - 2
+                  - 3
+                  format: int64
+                  type: integer
+                errorMessage:
+                  type: string
+              type: object
+            pendingContext:
+              properties:
+                jobId:
+                  description: JID of the Salt job in progress.
+                  type: string
+                size:
+                  description: Size of the volume.
+                  type: string
+                step:
+                  description: Name of the current step.
+                  enum:
+                  - PreparationStarted
+                  - PrepareVolume
+                  - GetVolumeSize
+                  - CreatePersistentVolume
+                  type: string
+              type: object
+            phase:
+              enum:
```

Comment:
Consider reordering these in the order they're supposed to go through?

Reply:
Done (I'll put `Failed` as first though, as it doesn't really have an order regarding the others):

- Failed
- Pending
- Available
- Terminating